src/schema.js

```diff
@@ -107,6 +107,11 @@ const blocks = {
 		examples: ['api-proxy', 'custom-requests', 'multiple-requests'],
 		labels: [null, DEFAULT_LABEL]
 	},
+	rate_limit: {
+		parents: ['backend'],
```

Comment:
`rate_limit` shall be only available in `backed` within `definitions`. We lack a dynamic `parents` property here (like `labels`) to make that possible.